aries-site/src/pages/templates/empty-state.mdx

```diff
@@ -0,0 +1,130 @@
+import { Example } from '../../layouts';
+import {
+  ActionEmptyState,
+  CardEmptyState,
+  EmptyStateAnatomy,
+  EmptyStateExample,
+  ErrorManagementEmptyState,
+  LayerEmptyState,
+  PageEmptyState,
+  NoData,
+} from '../../examples';
+import { Checkmark } from 'grommet-icons';
+
+<Example code="#" figma="#">
+  <EmptyStateExample />
+</Example>
+
+## When to use
+
+Common empty state use cases include:
+
+- “Day 0” use cases when a user first starts using an application
+- When the user completes an action that returns no results
+- When an error prevents the system from returning any data
```

Comment:
System-generated, task-generated, or both kinds of error? I'm a little confused on when to use notifications and when to use empty states here or whether or not there are opportunities to use both.

Reply:
Can you draft what you might expect this to say?

Reply:
@KennyAtHPE , you bring up a good point about when do I use empty state vs. inline notifications?

I think there is an aspect to having:
1. no data, where the empty state is now taking the place of where content would normally be displayed. Whereas the inline notification is likely supplementing/augmenting content where some data exists.
1. Would generally be system-level messages, but a user action could also lead to an empty state (e.g. too strict of filter or search criteria).

Reply:
@halocline I think generally those 2 points are a good distinction between inline notifications and empty states.

What I'm drawn most to within the current guidance is the use case "When an error prevents the system from returning any data." I feel as though I could see this exact bullet point under inline notifications. I'm concerned that designers would have an empty state and an inline notification that would both display the same message and any anchors/actions that would point to the same thing.

If these errors are user-generated, I would lean towards using inline notifications since it's a bit more intrusive and can better capture the user's attention. On the other hand, if it's a system-generated error, I could see an empty state being used instead. In both cases, I can see both being used together so it's a bit tricky depending on the use case.

Long story short though, for the time being, I would change that bullet point into something like this:
- When an error prevents the system from returning any data
      - For user-generated errors, [Inline Notifications](url) can be used together with empty states to provide additional feedback related to the user's actions.
      
I'd probably make a follow-up ticket to expand on this guidance and to provide visual examples as well.